ci_config.json

```diff
@@ -159,14 +159,6 @@
       "gettext"
     ]
   },
-  "imgui": {
-    "debian_packages": [
```

Comment:
Why are these removed?

Reply:
Not needed.

Reply:
Ehm, yes but they provide a useful test* for the CI? Please add them back...

* i.e. that the dependency lookup and compilation of the files still work

Reply:
IMO It's more useful to test other wraps.

Maybe edit CI to also test --wrap-mode=forcefallback

Reply:
Ah, fair, glfw and sdl2 are also available as wrap. allegro5 is not though.